backend/jobs/src/main/kotlin/com/linkedout/jobs/service/CompanyService.kt

```diff
@@ -0,0 +1,24 @@
+package com.linkedout.jobs.service
+
+import com.linkedout.jobs.dto.JobWithCategory
+import com.linkedout.jobs.model.Company
+import com.linkedout.jobs.repository.CompanyRepository
+import com.linkedout.jobs.repository.JobRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import java.util.UUID
+
+@Service
+class CompanyService(
+    @Autowired private val companyRepository: CompanyRepository
```

Comment:
This annotation should not be necessary.

```suggestion
    private val companyRepository: CompanyRepository
```